app/controllers/lrs_xapi_proxy_controller.rb

```
@@ -23,5 +24,7 @@ def xAPI
       render plain: response_body, status: response.code, content_type: content_type and return if content_type == LrsXapiProxy::OCTET_STREAM_MIME_TYPE
       render json: response_body, status: response.code and return if content_type == LrsXapiProxy::JSON_MIME_TYPE
     end
+  rescue RestClient::Exception => e
```

When we let the exception propagate, Rails created a HTML page response which was massive and blew up the content of the Honeycomb span sent from JS with the error details about the Ajax call that failed. Render as json since this is an API not a user rendered page.